README.md

```diff
@@ -75,6 +75,13 @@ By default it uses the default `tsc` command with the `--noEmit` flag to avoid g
 {
   flags = "--noEmit",
   spinner = { "⣾", "⣽", "⣻", "⢿", "⡿", "⣟", "⣯", "⣷" },
+  notify = {
+    enable = true,
```

Comment:
Lets rename this something like `enable_progress_notifications` or `enable_progress_status`. My rational for this is that it would be confusing to set `enable` to `false` and still get notifications from the plugin.

e.g.

```lua
  if vim.fn.executable(cmd) == 0 then
    vim.notify(
      format_notification_msg(
        "tsc was not available or found in your node_modules or $PATH. Please run install and try again."
      ),
      vim.log.levels.ERROR,
      { title = "TSC" }
    )

    return
  end

  if is_running then
    vim.notify(format_notification_msg("Type-checking already in progress"), vim.log.levels.WARN, { title = "TSC" })
    return
  end
```